apps/nextjs-website/src/_contents/translations.ts

```diff
@@ -309,6 +309,13 @@ export const translations = {
       },
     },
   },
+  webinar: {
+    whyParticipate: 'Perché partecipare?',
+    speakers: 'Speaker',
+    subscribe: 'Iscriviti',
+    relatedLinksTitle: 'Link utili',
+    speakersTitle: 'Chi Parla',
+  },
   auth: {
```

Comment:
I that translation for `auth` section are not yet removed like suggested in this [task DEV-1194](https://pagopa.atlassian.net/jira/software/projects/DEV/boards/280?selectedIssue=DEV-1194&text=DEV-1194).